packages/b/bell-ui.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "bell-ui",
+  "filename": "bell-ui.min.js",
+  "description": "UI component library for yox.js",
+  "keywords": [
+    "yox",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/divawth/bell-ui.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "bell-ui",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
👋 We try to avoid using open glob stars in auto-update configurations as it can lead to us including files we're not expecting, overwhelming the CDN.

Please can you update this to be more specific globs for the files that are actually needed.